vcpkg/concepts/supported-hosts.md

```diff
@@ -117,6 +121,12 @@ one of the above distros. As of this writing, those dependency versions and the
 
 ### Windows
 
+* Windows 7 / Server 2008 R2 and later
+
+`vcpkg.exe` should work on Windows 7 flavors, but the vast majority of the curated registry requires MSYS components
+like `pkg-config` which
+[no longer support versions of Windows older than 8.1](https://www.msys2.org/docs/windows_support/#current-policy).

```

Comment:
Don't sound like blaming msys2: Vcpkg team decided to choose the updated runtime.
(There is a successful CI run for the msy2-runtime-3.4 which remains compatible, and for experts, there is a switching variable.)

Reply:
BTW it is really "MSYS2".

Reply:
>Don't sound like blaming msys2: Vcpkg team decided to choose the updated runtime.

It was MSYS's maintainers decision that forced our hand here though. The alternative would have been to become MSYS maintainers ourselves and try to support old versions forever, which we do not have the capacity to do.

The point is, we are not going to break vcpkg.exe on those older platforms on purpose and if someone decides to replace the msys parts with ones that work on the older platforms, that's expected to work.

Reply:
My msys update PR initially came with the non-default MSYS2-maintained runtime which supports older Windows. It was a CR request from Microsoft to choose the default MSYS2-maintained runtime. Admittedly I don't know if there were other limitations. I have no machine with an old Windows to test this ATM.

Anyways, you accepted a switch, so users are not lost. If it works and and it they would know.
https://github.com/microsoft/vcpkg/blob/1cdb5e0b24c303b41fcac0dce00baf21881f6da2/scripts/cmake/vcpkg_acquire_msys.cmake#L436-L447

(Still waiting for someone to actively ask.)